parm/snow/obs/config/bufr_sfcsno_mapping.yaml

```diff
@@ -26,11 +25,8 @@ bufr:
       query: "[*/SNWSQ1/TOSD, */MTRMSC/TOSD, */STGDSNDM/TOSD]"
       transforms:
         - scale: 1000.0
-  filters:
-    - bounding:
-        variable: totalSnowDepth
-        lowerBound: 0
-        upperBound: 10000000
```

Comment:
For my understanding: 
- is this introducing a 10 m limit on snow depth obs?
- if they are above 10 m, are they discarded, or set to 10.?

Reply:
I previously put this large number (10000m) here for removing the missing values. Here, we don't need this any more because we need to set the missing snod values to 0 when sogr satisfies the defined conditions.

Reply:
We might still have missing values though wouldn't we?

Reply:
The missing values will be removed after applying sogr conditions.